DEVELOPMENT.md

```diff
@@ -37,7 +34,17 @@ development.
 1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
    managing development environments.
 
-### Environment setup
+### Create a cluster and a repo
+
+1. [Set up a kubernetes cluster](./docs/creating-a-kubernetes-cluster.md). NB:
+   do *not* install Istio or Knative using the instructions in that page, simply
+   create the cluster and come back here. However, if you _did_ install
+   Istio/Knative by mistake, just [clean up](#clean-up) your cluster and try
```

Comment:
I'm not sure that we need to prevent the Istio and/or knative install... it should be fine to simply re-apply over the existing install.

Reply:
Done